WalletWasabi.Fluent/ViewModels/CoinControl/Core/CoinCoinControlItemViewModel.cs

```diff
@@ -0,0 +1,21 @@
+using System.Collections.Immutable;
+using WalletWasabi.Blockchain.TransactionOutputs;
+
+namespace WalletWasabi.Fluent.ViewModels.CoinControl.Core;
+
+internal class CoinCoinControlItemViewModel : CoinControlItemViewModelBase
+{
+	public CoinCoinControlItemViewModel(SmartCoin smartCoin)
+	{
+		Amount = smartCoin.Amount;
+		IsConfirmed = smartCoin.Confirmed;
+		IsBanned = smartCoin.IsBanned;
+		IsCoinjoining = smartCoin.CoinJoinInProgress;
+		ConfirmationStatus = smartCoin.Confirmed ? "Confirmed" : "Pending confirmation";
```

Comment:
This is not correct.

In the View there is two different control for indicating `confirmed` and `pending`.
`ConfirmationStatus` is only displayed for confirmed.

For consistency, it should display the number of confirmations.
```suggestion
		ConfirmationStatus = $"{smartCoin.Height} confirmation{TextHelpers.AddSIfPlural(smartCoin.Height)}";
```